app/src/main/java/com/capstone/event_finder/network/EventViewModel.java

```diff
@@ -20,9 +20,7 @@ public class EventViewModel extends AndroidViewModel {
 
     private final EventRepository eventRepository;
     private final EventApi eventApi;
-    private final List<Bookmark> userBookmarks = new ArrayList<>();
     private final MutableLiveData<List<Event>> bookmarkList;
-    private final ArrayList<String> bookmarkIds = new ArrayList<>();
```

Comment:
✅ Thanks for making this change, it might seem trivial but will help avoid troubles in the future.

The "Clean Code" book I recommended has more information on this under "Chapter 3: Functions" under "Have No Side Effects".